### PR TITLE
Adapt chectl to new plugin notation

### PR DESCRIPTION
**Pull request by garagatyi**
Do not merge before https://github.com/eclipse/che/pull/13204 gets merged

### CODE REVIEW

File: test/api/requests/devfile.invalid [hunk 1]
```diff
@@ -8,10 +8,10 @@ projects:
 tools:
   - name: theia-ide
```
Comment (garagatyi) [inline test/api/requests/devfile.invalid]: @metlos isn't this name shouldn't be here anymore?

Comment (l0rd) [reply test/api/requests/devfile.invalid]: Good catch @garagatyi

**Comment by garagatyi** [comment]
I've merged https://github.com/eclipse/che/pull/13204 but since chectl depends on Che wsmaster nightly image we can merge it tomorrow.
@l0rd can you leave you review, so I would be able to proceed with it?

File: test/api/requests/devfile.invalid (hunk 1 above)
Comment (l0rd) [reply test/api/requests/devfile.invalid]: @garagatyi since you have spotted this can you please remove it?

Comment (garagatyi) [reply test/api/requests/devfile.invalid]: sure

Comment (garagatyi) [reply test/api/requests/devfile.invalid]: done

**Comment by garagatyi** [comment]
@l0rd just checked it locally - workspace creation/start works as expected with the current nightly.
I'm ready to merge the PR as soon as you approve it

**Pull request merged**